.gitignore

```diff
@@ -97,6 +97,8 @@ scikits
 !np_datetime_strings.c
 *.c
 *.cpp
+# Unignore source c files
+!pandas/_libs/src/**/*.c
```

Comment:
The np_datetime.c and np_datetime_strings.c above are in pandas/_libs/tslibs/src

maybe would could include .c files under any directory named `src` in the `_libs` tree

Reply:
I removed the np lines since they are caught by my new line. The line I added will ignore all c files under src/ so we're good there.

Reply:
I think it should be `!pandas/_libs/**/src/**/*.c` so that any `.c` files added to the `pandas/_libs/tslibs/src` are included in the future.